entity-service-client/build.gradle.kts

```diff
@@ -13,7 +13,7 @@ dependencies {
   api(project(":entity-service-api"))
   api("com.typesafe:config:1.4.1")
 
-  implementation("org.hypertrace.core.grpcutils:grpc-client-utils:0.7.3")
+  implementation("org.hypertrace.core.grpcutils:grpc-client-utils:0.11.2")
```

Comment:
please upgrade grpc consistently across the service.

Reply:
added

Reply:
moved to grpc `1.50.0`